example/alertmanager-crd-conversion/patch.json

```diff
@@ -3,8 +3,7 @@
    "kind": "CustomResourceDefinition",
    "metadata": {
       "annotations": {
-         "controller-gen.kubebuilder.io/version": "v0.16.4",
-         "operator.prometheus.io/version": "0.77.1"
```

Comment:
This is not expected. The `operator.prometheus.io/version` annotation needs to be preserved for all CRDs.